portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,109 @@
+let answer = -1;
```

Comment:
Suggest renaming this variable to ... "correctAnswer", maybe.

Reply:
renamed